atv/settings.py

```diff
@@ -193,6 +197,15 @@
 if not DEBUG and DEBUG_FIELD_ENCRYPTION_KEY in FIELD_ENCRYPTION_KEYS:
     raise ImproperlyConfigured("Cannot use the debug encryption key in production")
 
+# API Documentation
+ENABLE_SWAGGER_UI = env.bool("ENABLE_SWAGGER_UI")
```

Comment:
You can also use `env("ENABLE_SWAGGER_UI")` here since you've defined this env variable as a boolean above.

Reply:
Ahh cool! I thought you had to parse it when retrieving it, but let's then skip this 👍